sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -201,15 +205,24 @@ const eventHubConsumerClient = new EventHubConsumerClient(consumerGroupName, ehC
 
 const subscription = eventHubConsumerClient.subscribe(
   partitionId, {
-    // In V5 we deliver messages in batches, rather than a single message 
-    // at a time. You can control the batch size via the options passed to the client.
-    processEvents: (messages, context) => {},
+    // In V5 we deliver events in batches, rather than a single message at a time.
+    // You can control the batch size via the options passed to the client.
+    //
+    // If you have asynchronous code running in your callback, it will be awaited before the
+    // callback is called for the next batch of events. 
+    processEvents: (events, context) => { /** your code here **/ },
 
     // Prior to V5 errors were handled by separate callbacks depending 
     // on where they were thrown i.e when managing different partitions vs receiving from each partition.
     // 
     // In V5 you only need a single error handler for all of those cases.
-    processError: onErrorHandler
+    processError: (error, context) => { 
```

Comment:
Do you see any value in calling out out that this can be an async function like we did for `processEvents`? Certainly seems less important but asking for consistency's sake.

Reply:
No, my hope is for API reference docs to do that

Reply:
The call out for processEvents was because that change is one of our biggest selling points based on the feedback we have been getting